tests/Oryx.Integration.Tests/NodeEndToEndTests.cs

```diff
@@ -34,6 +34,7 @@ public NodeEndToEndTestsBase(ITestOutputHelper output, TestTempDirTestFixture te
             DockerVolume.CreateMirror(Path.Combine(_hostSamplesDir, "nodejs", appName));
     }
 
+    [Trait("category", "node")]
```

Comment:
is there a way to run tests in a "catch-all" fashion, that is, those not in any category? This is to avoid not running tests that aren't categorized, like this case